core/src/main/java/org/bptlab/cepta/utils/converters/PlannedTrainDataDatabaseConverter.java

```diff
@@ -15,18 +17,22 @@ public PlannedTrainData fromResult(ResultSet result) throws Exception {
       event.setId(result.getInt("id"));
       event.setTrainId(result.getInt("train_id"));
       event.setLocationId(result.getInt("location_id"));
-      convertTimestamp(result.getTimestamp("planned_time"), event::setPlannedTime);
```

Comment:
Werden die ganzen Methoden convert... die in dem File an vielen Stellen stehen noch benötigt?

Reply:
jo kommt raus